actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BulkUploadManagementActor.java

```diff
@@ -134,9 +138,12 @@ private void getUploadStatus(Request actorMessage) {
       Map<String, Object> resMap = resList.get(0);
       String objectType = (String) resMap.get(JsonKey.OBJECT_TYPE);
       if ((int) resMap.get(JsonKey.STATUS) == ProjectUtil.BulkProcessStatus.COMPLETED.getValue()) {
+        resMap.put(JsonKey.PROCESS_ID, resMap.get(JsonKey.ID));
```

Comment:
Write a private method which updates status and message in response map.

updateResponseStatus(response, status)